packages/im-components/src/atoms/button/constants/button.sizes.js

```diff
@@ -1,6 +1,6 @@
-const SMALL = "SMALL";
-const MEDIUM = "MEDIUM";
-const LARGE = "LARGE";
+const SMALL = "small";
+const MEDIUM = "middle";
```

Comment:
Is it "middle" or "default"? I tried with "default" for Medium string.

Reply:
If you check size here you can see it has three options available (small,middle,large) 
default is middle
https://ant.design/components/button/#API